<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Monitor</class>
    <widget class="QWidget" name="Monitor">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>849</width>
                <height>578</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Monitor</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
                <layout class="QVBoxLayout" name="verticalLayout">
                    <item>
                        <widget class="QPushButton" name="pushButton_add_variable">
                            <property name="text">
                                <string>添加变量</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <layout class="QFormLayout" name="formLayout">
                            <item row="0" column="1">
                                <widget class="QComboBox" name="comboBox_monitor_var">
                                    <item>
                                        <property name="text">
                                            <string>正弦波</string>
                                        </property>
                                    </item>
                                    <item>
                                        <property name="text">
                                            <string>方波</string>
                                        </property>
                                    </item>
                                </widget>
                            </item>
                            <item row="1" column="1">
                                <widget class="QComboBox" name="comboBox_line_type">
                                    <item>
                                        <property name="text">
                                            <string>直线</string>
                                        </property>
                                    </item>
                                    <item>
                                        <property name="text">
                                            <string>左侧步长</string>
                                        </property>
                                    </item>
                                    <item>
                                        <property name="text">
                                            <string>右侧步长</string>
                                        </property>
                                    </item>
                                    <item>
                                        <property name="text">
                                            <string>中间步长</string>
                                        </property>
                                    </item>
                                    <item>
                                        <property name="text">
                                            <string>脉冲</string>
                                        </property>
                                    </item>
                                    <item>
                                        <property name="text">
                                            <string />
                                        </property>
                                    </item>
                                </widget>
                            </item>
                            <item row="2" column="1">
                                <widget class="QComboBox" name="comboBox_dot_type">
                                    <item>
                                        <property name="text">
                                            <string>无</string>
                                        </property>
                                    </item>
                                    <item>
                                        <property name="text">
                                            <string>单点</string>
                                        </property>
                                    </item>
                                    <item>
                                        <property name="text">
                                            <string>十字</string>
                                        </property>
                                    </item>
                                    <item>
                                        <property name="text">
                                            <string>圆形</string>
                                        </property>
                                    </item>
                                    <item>
                                        <property name="text">
                                            <string>星形</string>
                                        </property>
                                    </item>
                                    <item>
                                        <property name="text">
                                            <string>三角形</string>
                                        </property>
                                    </item>
                                </widget>
                            </item>
                            <item row="3" column="1">
                                <widget class="QComboBox" name="comboBox_line_color">
                                    <item>
                                        <property name="text">
                                            <string>红</string>
                                        </property>
                                    </item>
                                    <item>
                                        <property name="text">
                                            <string>绿</string>
                                        </property>
                                    </item>
                                    <item>
                                        <property name="text">
                                            <string>蓝</string>
                                        </property>
                                    </item>
                                </widget>
                            </item>
                            <item row="0" column="0">
                                <widget class="QLabel" name="label_monitor_variable">
                                    <property name="text">
                                        <string>变量名称</string>
                                    </property>
                                </widget>
                            </item>
                            <item row="1" column="0">
                                <widget class="QLabel" name="label_line_type">
                                    <property name="text">
                                        <string>线条类型</string>
                                    </property>
                                </widget>
                            </item>
                            <item row="2" column="0">
                                <widget class="QLabel" name="label_dot_type">
                                    <property name="text">
                                        <string>点类型</string>
                                    </property>
                                </widget>
                            </item>
                            <item row="3" column="0">
                                <widget class="QLabel" name="label_line_color">
                                    <property name="text">
                                        <string>线条颜色</string>
                                    </property>
                                </widget>
                            </item>
                            <item row="4" column="0">
                                <widget class="QLabel" name="label_line_width">
                                    <property name="text">
                                        <string>线宽</string>
                                    </property>
                                </widget>
                            </item>
                            <item row="4" column="1">
                                <widget class="QSlider" name="horizontalSlider_line_width">
                                    <property name="orientation">
                                        <enum>Qt::Horizontal</enum>
                                    </property>
                                </widget>
                            </item>
                            <item row="5" column="0">
                                <widget class="QComboBox" name="comboBox_exist_variable" />
                            </item>
                            <item row="5" column="1">
                                <widget class="QPushButton" name="pushButton_remove_variable">
                                    <property name="text">
                                        <string>删除变量</string>
                                    </property>
                                </widget>
                            </item>
                            <item row="6" column="1">
                                <spacer name="horizontalSpacer_2">
                                    <property name="orientation">
                                        <enum>Qt::Horizontal</enum>
                                    </property>
                                    <property name="sizeHint" stdset="0">
                                        <size>
                                            <width>40</width>
                                            <height>20</height>
                                        </size>
                                    </property>
                                </spacer>
                            </item>
                            <item row="6" column="0">
                                <spacer name="horizontalSpacer">
                                    <property name="orientation">
                                        <enum>Qt::Horizontal</enum>
                                    </property>
                                    <property name="sizeHint" stdset="0">
                                        <size>
                                            <width>40</width>
                                            <height>20</height>
                                        </size>
                                    </property>
                                </spacer>
                            </item>
                            <item row="7" column="1">
                                <layout class="QFormLayout" name="formLayout_parameter" />
                            </item>
                            <item row="7" column="0">
                                <widget class="QComboBox" name="comboBox_sign_type">
                                    <item>
                                        <property name="text">
                                            <string>正弦波</string>
                                        </property>
                                    </item>
                                    <item>
                                        <property name="text">
                                            <string>方波</string>
                                        </property>
                                    </item>
                                    <item>
                                        <property name="text">
                                            <string>梯形波</string>
                                        </property>
                                    </item>
                                </widget>
                            </item>
                        </layout>
                    </item>
                </layout>
            </item>
        </layout>
    </widget>
    <resources />
    <connections />
</ui>